sympy/__init__.py

```diff
@@ -22,6 +22,21 @@
 
 from sympy.release import __version__
 
+if 'dev' in __version__:
+    def enable_warnings():
+        import os
+        import warnings
+        action ='default'
+        travis = (os.environ.get('TRAVIS','') == 'true')
```

Comment:
Any simple way to know we are running the tests ? setting `default` by default seem useless as test will never fails, and setting `error` seem annoying for local user where Sympy would not even import.